docs/post.sh

```diff
@@ -0,0 +1 @@
+#!/usr/bin/env bash
```

Comment:
Why is this needed?

Reply:
I brought this in from the sdk. @fadeev is it required?

Reply:
its an empty script file.

Reply:
Nope, not required.